packages/chart/src/helpers/isConvertLineToBarGraph.ts

```diff
@@ -1,4 +1,13 @@
-export const isConvertLineToBarGraph = (visualizationType, filteredData, allowLineToBarGraph) => {
-  const convertLineToBarGraph = visualizationType === 'Line' && filteredData?.length < 3 && allowLineToBarGraph ? true : false
+export const isConvertLineToBarGraph = (
+  visualizationType,
+  filteredData,
+  allowLineToBarGraph,
+  isDynamicWithLessThanThreePoints = false
+) => {
+  const convertLineToBarGraph =
+    ((visualizationType === 'Line' && filteredData?.length < 3) || isDynamicWithLessThanThreePoints) &&
+    allowLineToBarGraph
+      ? true
+      : false
```

Comment:
clean this up: 


if(!allowLineToBarGraph) return false;
lineWithLessThanThreePoints = visualizationType === 'Line' && filteredData?.length < 3

return lineWithLessThanThreePoints || isDynamicWithLessThanThreePoints

// also move the logic for calculating the isDynamicWithLessThanThreePoints into this function